pages/blog/blog-post-template.html

```diff
@@ -29,6 +29,26 @@ <h3>Placeholder Section Title</h3>
         <p>This is a placeholder paragraph. You should replace it with your own blog post text. If you need more paragraphs, add more p tags below this one. If you need more sections, add more h3 tags below this one.</p>
         <footer-component></footer-component>
     </body>
+
+    <!-- To do: add css styling. align header caption. fill out head section with SEO etc. make navbar and footer work. make social links pretty-->
+
+    <!-- WEBSITE SCRIPTS -->
+    <!-- script below will import all functions we already use on the rest of the website -->
+    <script src="./assets/js/homepage-scripts.js"></script>
+
+    <!-- script below will import/map all our HTML components, without it none of them will be visible -->
+    <script src="/assets/js/components/component-map.js" type="module"defer></script>
+
+    <!-- script below is used for google analytics-->
+    <script>
+        window.dataLayer = window.dataLayer || [];
+
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+        
+        gtag("js", new Date());
+        gtag("config", "G-E7KQ4C9DKP");
+    </script>
```

Comment:
Hiya @john-fowle 😄 I thought I'd help out by bringing our scripts over and just pointing out what each of them does